aqt/archives.py

```diff
@@ -780,17 +780,19 @@ class ToolArchives(QtArchives):
     def __init__(
         self,
         os_name: str,
+        target: str,
         tool_name: str,
         base: str,
         arch: Optional[str] = None,
         timeout: Tuple[int, int] = (5, 5),
     ):
+        self.target = target
         self.tool_name = tool_name
         self.os_name = os_name
```

Comment:
I have a question about the data members `ToolArchives.target` and `ToolArchives.os_name`. The parent class already has data members `target` and `os_name`. Are these duplicates? Is there a reason for keeping both copies?

Reply:
Thank you. Fixed.

Reply:
No problem, but I think you removed `self.tool_name` by mistake; that one still needs to be in ToolArchives